content/pl/training/_index.html

```diff
@@ -112,7 +112,6 @@ <h2>Partnerzy szkoleniowi Kubernetes</h2>
     </center>
   </div>
   <div class="main-section landscape-section">
-    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
-    <script src="https://landscape.cncf.io/iframeResizer.js"></script>
+    {{< cncf-landscape helpers=false category="kubernetes-training-partner" >}}
```

Comment:
```suggestion
    {{< cncf-landscape helpers="false" category="kubernetes-training-partner" >}}
```

Reply:
No, I tested in the en version when I did the PR, there was an error
https://github.com/kubernetes/website/pull/24014

Reply:
huh ;D